docs/release-notes.mdx

```diff
@@ -296,8 +295,8 @@ can be significantly cheaper in L1 ETH costs depending on market demand.
 
     EIP-4844's arrival on Ethereum Mainnet enabled Ethereum L2s to start using a new transaction type 
     that transports L2 transaction and messaging data to L1 in "blobs". Blobs are a considerably 
-    more cost-efficient alternative to the existing method of posting L2 transaction and messaging data 
-    to L1, which involved sending `calldata` embedded in a transaction.
+    more cost-efficient alternative to the existing method of posting L2 transactions and messaging data 
```

Comment:
```suggestion
    more cost-efficient alternative to the existing method of posting L2 transaction and messaging data 
```

`transaction data + messaging data`, rather than `transactions + messaging data`